### PR TITLE
Move LowPriorityImplicits to a com.softwaremill.quicklens to avoid conflicts with other softwaremill libraries

### LINKED ISSUE #237: LowPriorityImplicits is in package com.softwaremill
Hello,

It should be in the `com.softwaremill.quicklens` package to make sure it doesn't clash with any other class with the same name (LowPriorityImplicits is very generic) in any of your other libraries/projects.

Regards

### PR DESCRIPTION
**Pull request by KacperFKorban**
closes #237

### CODE REVIEW

File: quicklens/src/main/scala-2.13+/com.softwaremill.quicklens/LowPriorityImplicits.scala [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+   * `QuicklensEach` is in `LowPriorityImplicits` to not conflict with the `QuicklensMapAtFunctor` on `each` calls.
+   */
+  implicit class QuicklensEach[F[_], T](t: F[T])(implicit f: QuicklensFunctor[F, T]) {
+    @compileTimeOnly(canOnlyBeUsedInsideModify("each"))
+    def each: T = sys.error("")
+
+    @compileTimeOnly(canOnlyBeUsedInsideModify("eachWhere"))
+    def eachWhere(p: T => Boolean): T = sys.error("")
+  }
+}
```
Comment (slandelle) [inline quicklens/src/main/scala-2.13+/com.softwaremill.quicklens/LowPriorityImplicits.scala:17]: Missing EOL

**Review by slandelle — CHANGES_REQUESTED** [review]
One small nit.
Thanks!

**Review by slandelle — CHANGES_REQUESTED** [review]
LGTM

It might be cleaner fix the `com.softwaremill.quicklens` folder into a proper `com/softwaremill/quicklens` hierarchy.

**Review by slandelle — APPROVED** [review]

**Comment by adamw** [comment]
Thanks!

**Pull request merged**